contracts/modules/LoanClosingsBase.sol

```diff
@@ -361,11 +361,10 @@ contract LoanClosingsBase is LoanClosingsShared, LiquidationHelper {
 				// 1. pay back the excess reward to the lender
 				// 2. pay the complete collateral to msg.sender
 				// 3. close the position & emit close event
```

Comment:
misleading comments not corresponding to the fixed logic

Reply:
This 3 comments is implemented in the `_closeWithSwap` function